test/controllers/dit_landing_page_controller_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+describe DitLandingPageController do
+  describe "GET show" do
+    it "renders the page for the en locale" do
+      get :show
+      assert_response :success
+      assert_select "h1", "Trade with the UK from 1 January 2021 as a business based in the EU"
+    end
+  end
+end
```

Comment:
I'd love a few more tests - particularly covering where there's logic, or we're relying on the structure of the yaml to be able to iterate maybe.

Reply:
Yes fair point! Will work on those on Monday.